utils/http.go

```diff
@@ -5,6 +5,7 @@
 package utils
 
 import (
+	goMime "mime"
```

Comment:
Please don't use import-aliases for builtin packages. Better rename the variables below to something more or less meaningful.

Reply:
@leonklingele thanks for reviewing. done